strongbox-security/strongbox-security-api/src/test/java/org/carlspring/strongbox/security/certificates/KeyStoreManagerIntegrationTestIT.java

```diff
@@ -161,7 +161,7 @@ public void testSocksProxy()
         assertTrue(certs.isEmpty());
     }
 
-    @Ignore
+    @Disabled
```

Comment:
@fuss86 / @sbespalov,

Do you remember the reason why these were `@Ignored`? Do we have follow-up tasks for them?